client/js/lounge.js

```diff
@@ -448,6 +452,12 @@ $(function() {
 				.find(".unread-marker")
 				.appendTo(container);
 		}
+		if (data.msg.type === "channel_list") {
+			// Delay by 1 second so the browser has time to render all channels
+			$(target + " .chat").stop().delay(1000).animate({
+				scrollTop: 0
+			});
+		}
```

Comment:
@swordbeta, I believe this would be as simple as callin `unsticky()` like [here](https://github.com/thelounge/lounge/blob/master/client/js/lounge.js#L1021), but that needs testing as I haven't played with this myself yet.